other/edge-server/client/README.md

```diff
@@ -1,38 +1,116 @@
 # Edge Server with MongoDB Node.js Driver
 
-This example application uses the MongoDB Node.js Driver to connect to 
-Edge Server. It has a React frontend for performing CRUD operations. It uses 
-the MongoDB Node.js Driver in an Express server application to connect to
-the Edge Server.
+This example todo application demonstrates using Edge Server with a MongoDB Driver. It uses the MongoDB Node.js Driver in an Express server application to connect to 
+Edge Server. It has a React frontend for performing CRUD operations. 
 
 React client -> Express server (Node.js Driver) -> Edge Server -> Atlas
 
 To use this example application, you must:
 
-1. Create an App Services App using the backend configuration provided in
-  this project's [TODO: Cory insert directory name here] directory.
+1. Create an App Services App that uses the configured backend provided in 
+  this project's ../backend directory.
 2. Download, configure, and start the Edge Server.
 3. Install dependencies and start the Node.js Express server.
 4. Install dependencies and start the React server.
 
-This is a MERN stack that connects to the Edge Server instead of Atlas. For
-a MERN stack tutorial, refer to 
+This is a MERN stack that connects to the Edge Server instead of Atlas. 
+(Note that MongoDB wire protocol connections connect directly to Edge Server 
+without authorization.) 
```

Comment:
Adding authorization for wireprotocol connections is part of the Public Preview, so we should probably be more explicit here. We can update this app once authentication for wireprotocol connections is supported by Edge Server.

```suggestion
In this version of the example app, MongoDB wire protocol connections connect directly to Edge Server without authorization. 
```